sqlserver/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - sqlserver
 
+## 9.0.1 / 2022-10-10
+
+* [Fixed] sqlserver/connection: always cast port to str. See [#13061](https://github.com/DataDog/integrations-core/pull/13061).
```

Comment:
This could use some love as a changelog entry that's more user-oriented as these are public facing 🤔

Reply:
```suggestion
* [Fixed] Always cast the configured port to as string. See [#13061](https://github.com/DataDog/integrations-core/pull/13061).
```

What do you think?

Reply:
Yeah, that would definitely be an improvement. I don't have the context for the fix so I don't know what it did exactly, but I'd personally try to look at it from the user's perspective, which after a (very) quick look, it may very well be something along the lines of

```suggestion
* [Fixed] Fix support for setting the port as a separate config option. See [#13061](https://github.com/DataDog/integrations-core/pull/13061).
```

Reply:
It was already a separate config option, but we allow the user to configure it as a int or a string (i.e '22' or 22 in the config yaml file)

So maybe something like this

```suggestion
* [Fixed] Allow users to configure the port as a int or as a string. See [#13061](https://github.com/DataDog/integrations-core/pull/13061).
```

Reply:
It works for me.